workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -101,3 +101,12 @@ def write_chunks(config: UserConfig):
     chunks = get_chunks(diagnostic_file_configs)
     with open("chunks.yaml", "w") as f:
         yaml.safe_dump(chunks, f)
+
+
+def get_model_urls(config_dict: dict) -> List[str]:
```

Comment:
This function is the iffy-est part of the PR to me as it couples the metadata to the structure of our config object. I'm open to hearing if there is another alternative to get these dependencies.

Reply:
I think I agree that this location makes sense for this to live here since it's close to the config.  I think we just have to get the return type right so it's generally useful.  I'm thinking maintaining hierarchy would probably be better?  The empty container vs. None probably pertains here too on whether one is easier to query/filter.

```
{
    scikit_learn: <str or None>,  
    prephysics: <mapping of unique items to url or None> 
}

Reply:
Yah, this is tricky, there's also possibly emulator/fine-res model dependencies. What about something more general... like search for all GCS urls in the config that have `"trained_models"` in them? (not really sure we want to do this... would only work for models trained via the argo workflows)

Reply:
>as it couples the metadata to the structure of our config object.
I think this is unavoidable since only the config object knows what it's "dependencies" are. 

I can think of a few general solutions that would be more robust than searching for a "hardcoded" string.
1. add a `.dependencies()` method to each class in our dataclass hiearchy that advertises its deps. This is a bit laborious and would still require code to traverse the config tree.
2. Require that all "step" level deps be handled as assets by fv3config. Call `fv3config.config_to_asset_list(config)` and simply return the urls in the asset list. I recently made `fv3config.config_to_asset_list` public API to enable this kind of approach.